openadapt/config.defaults.json

```diff
@@ -18,6 +18,7 @@
     "REPLAY_STRIP_ELEMENT_STATE": true,
     "RECORD_VIDEO": true,
     "RECORD_AUDIO": true,
+    "RECORD_BROWSER_EVENTS": true,
```

Comment:
Please default to false

Reply:
![image](https://github.com/user-attachments/assets/8eba24a3-7c68-4deb-8fbe-9d03cece1482)

Reply:
Done here: 388156c5202ce1b887fcf2f1109b2b751bf0bf19